appengine/gophers/gophers-6/index.html

```diff
@@ -36,21 +36,18 @@ <h1>The Gopher Network</h1>
 
 {{with .Notice}}<div id="notice">{{.}}</div>{{end}}
 
-<!-- [START form] -->
```

Comment:
gophers-6 is about to be published.

Reply:
Currently, region tags for gophers-6 are not used in any of the docs. @angelafunk please let me know if you are planning on using these region tags so we can get them on the samples tracker. Thanks!

Reply:
Yes! will be using region tags for gophers-6